prereise/gather/demanddata/bldg_electrification/floor_area_avg_slopes.py

```diff
@@ -0,0 +1,608 @@
+import os
+
+import geopandas as gpd
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from zone_profile_generator import zone_shp_overlay
+
+from prereise.gather.demanddata.bldg_electrification import const
+
+
+def get_zone_floor_area(iso):
+    """Computes the zone floor area for each ISO.
+
+    :param str iso: abbrev. name of ISO.
+    :return: (*pandas.DataFrame*) -- Floor area in square meters for all the zones with breakdowns of residential and commercial, total heating and total cooling
```

Comment:
We will need to break this line into smaller ones so that each of them is less or equal to 88 characters to follow PEP8.

Reply:
Sorry, but I had a query. When you mean breakdown into smaller lines, is it just splitting the sentence between the current line and the line below

Reply:
Yes, we are good if every line has length <= 88 characters (not sure whether you are using any IDE which can wrap columns automatically as needed)

Reply:
Made the changes

Reply:
Made the changes